### DISCUSSION TITLE
metacommunity

### DISCUSSION THREAD

**Discussion by mikatakahashi** [IBAHCM/RPiR#337] [Archive] [answered]
Hi @soniamitchell @richardreeve,

I've got a minor question. 
When I use metacommunity() from rdiversity package, it gives me 'Metacommunity matrix was normalised to sum to 1.' message but since I use it within a loop, it shows up a ton times in my report.... Do you know if I am doing something wrong or how not to show this message? Thank you!

**Comment by soniamitchell** [comment, marked as answer]
Hello @mikatakahashi, it’s just telling you that it’s normalising the metacommunity. If you normalise it yourself prior to generating the metacommunity object, then it won’t have to and you won’t get the message 👍

**Reply by mikatakahashi** [reply]
Got it! Thank you!
